environment/container/kubernetes/kubeconfig.go

```diff
@@ -42,6 +42,10 @@ func (c kubernetesRuntime) provisionKubeconfig(ctx context.Context) error {
 	})
 
 	kubeconfFile := filepath.Join(hostKubeDir, "config")
+	envKubeConfFile := c.host.Env("KUBECONFIG")
+	if envKubeConfFile != "" {
+		kubeconfFile = strings.Split(envKubeConfFile, ":")[0]
```

Comment:
To make this Windows-friendly, you could use `os.PathListSeparator` instead of `":"`

Reply:
While I agree that using `os.PathListSeparator` is better, I would really like to know why `Colima - container runtimes on **macOS (and Linux)** with minimal setup.` would need to be more windows-friendly? It does not run on Windows as it is a macOS and Linux **ONLY** project.

Reply:
You both have valid points. Considering that Colima is still far from being able to properly support windows, we do not have to worry about it.

However, using `os.PathListSeparator` is pretty much effortless.